extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -1498,4 +1499,24 @@ describes.realWin('additional amp-ad-network-doubleclick-impl',
           });
         });
       });
+
+      describe('#preconnect', () => {
+        beforeEach(() => {
+          element = createElementWithAttributes(doc, 'amp-ad', {
+            'width': '200',
+            'height': '50',
+            'type': 'doubleclick',
+          });
+          doc.body.appendChild(element);
+          impl = new AmpAdNetworkDoubleclickImpl(element);
+        });
+
+        it('should preload nameframe', () => {
+          const preloadSpy = sandbox.stub(Preconnect.prototype, 'preload');
+          expect(impl.getPreconnectUrls()).to.deep.equal(
+              ['https://securepubads.g.doubleclick.net/']);
```

Comment:
It would be nice to have a comment that we're also preconnecting to `tpc.googlesyndication.com` but that this happens as a side effect of calling `preload`.

Reply:
Done